### PR TITLE
add filter and column for recovery token usage

### PR DESCRIPTION
**Pull request by 0x46616c6b**
As Admin I want to see if a user has set a recovery token or not. Also I want to filter the users for this criteria.

### CODE REVIEW

File: src/Admin/UserAdmin.php [hunk 1]
```diff
@@ -152,6 +167,9 @@ protected function configureListFields(ListMapper $listMapper)
             ->addIdentifier('email')
             ->add('creationTime')
             ->add('updatedTime')
+            ->add('hasRecoverySecretBox', 'boolean', [
+                'label' => 'Recovery Token',
```
Comment (doobry-systemli) [inline src/Admin/UserAdmin.php]: I can imagine that this label causes the column to become very wide and take a lot of space? That would be unfortunate for a boolean column which has only 0/1 as values. Please ignore the comment if my assumption is wrong.

**Comment by 0x46616c6b** [comment]
screenshot_2019-03-05 admin - user list

File: src/Admin/UserAdmin.php (hunk 1 above)
Comment (0x46616c6b) [reply src/Admin/UserAdmin.php]: I added a screenshot in the comments here. So you can get a feeling for the visual change.

**Comment by t2d** [comment]
Why can I sort by every other column but not "Recovery Token"?

**Comment by 0x46616c6b** [comment]
screenshot_2019-03-05 admin - user list 2

**Comment by t2d** [comment]
I added some code to export the same information to Munin. Hope it's okay to do it with this PR. Very similar idea.

**Pull request merged**